react/webpack.config.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
```

Comment:
Make sure you update hound to acount for ES6 if you want to use it for JavaScript!